ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/isis-interfaces.j2

```diff
@@ -0,0 +1,44 @@
+{# isis interfaces #}
+{% if router_isis is defined and router_isis is not none %}
+### ISIS interfaces Summary
+
+| Interface | ISIS instance | ISIS metric | Interface mode | 
+| -------- | -------- | -------- | -------- | 
+{#- ethernet interfaces -#}
```

Comment:
As much as possible, we should avoid `rstrip`/`lstrip` as it changes default jinja2 behavior

Reply:
done